apps/vscode/editor/src/app.tsx

```diff
@@ -4,14 +4,44 @@ import { Tldraw, TldrawApp, TDFile, TDDocument } from '@tldraw/tldraw'
 import { vscode } from './utils/vscode'
 import { defaultDocument } from './utils/defaultDocument'
 import type { MessageFromExtension, MessageFromWebview } from './types'
+import {toSVG, fromSVG} from "./utils/svgEmbedder";
+import { throttle } from 'throttle-debounce';
+import type { MessageFromWebview } from '../types'
 
 // Will be placed in global scope by extension
+declare let svgEmbedded:boolean;
+declare let initialFileContent:string;
 declare let currentFile: TDFile
 
+function parseFile(fileContent):TDFile{
+  try {
+    if( svgEmbedded === false ){
```

Comment:
```suggestion
    if (!svgEmbedded) {
```

Reply:
I'm more of a fan of the explicit style one.

Reply:
I'm also not a fan of any styling 🗡️ 
But what if we have a standard format for everybody already?

https://github.com/tldraw/tldraw/blob/main/.prettierrc

Reply:
`svgEmbeded` is `boolean` so, we're not expecting to have a falsy value.
That's means to us, you're already checking your condition explicitly as your definition.